eboutic/views.py

```diff
@@ -192,62 +155,47 @@ def get_context_data(self, **kwargs):
         return kwargs
 
 
-class EbouticPayWithSith(TemplateView):
-    template_name = "eboutic/eboutic_payment_result.jinja"
-
-    def post(self, request, *args, **kwargs):
+@login_required
+@require_POST
+def pay_with_sith(request):
+    basket = Basket.from_session(request.session)
+    refilling = settings.SITH_COUNTER_PRODUCTTYPE_REFILLING
+    if basket is None or basket.items.filter(type_id=refilling).exists():
+        return redirect("eboutic:main")
+    c = Customer.objects.filter(user__id=basket.user.id).first()
+    if c is None:
+        return redirect("eboutic:main")
+    if c.amount < basket.get_total():
+        res = redirect("eboutic:payment_result", "failure")
```

Comment:
```suggestion
       return redirect("eboutic:payment_result", "failure")
```
Comme ça t'as pas besoin du else après et c'est plus propre

Reply:
Si, il y a besoin de supprimer le cookie après ça. Sinon quand l'utilisateur reviendra sur la boutique en ligne, il reverra exactement le panier pour lequel il vient de payer. C'est un coup à ce qu'il croit qu'il a payé pour rien, sinon. Bref, retirer la suppression du cookie, ça serait un mauvais choix d'UX.